samples/namespace-monitoring/README.md

```diff
@@ -65,24 +66,24 @@ kubectl label ns $VIYA_ONE_NS sas.com/viya-namespace=$VIYA_ONE_NS
 kubectl label ns $VIYA_TWO_NS sas.com/viya-namespace=$VIYA_TWO_NS
 ```
 
-7. Deploy cluster monitoring (including the Prometheus Operator) with a custom user directory and no SAS Viya dashboards.
+5. Deploy cluster monitoring (including the Prometheus Operator) with a custom user directory and no SAS Viya dashboards.
 ```bash
 VIYA_DASH=false monitoring/bin/deploy_monitoring_cluster.sh
 ```
 
-8. Deploy standard SAS Viya monitoring components each SAS Viya namespace.
+6. Deploy standard SAS Viya monitoring components each SAS Viya namespace.
 ```bash
 VIYA_NS=$VIYA_ONE_NS monitoring/bin/deploy_monitoring_viya.sh
 VIYA_NS=$VIYA_ONE_NS monitoring/bin/deploy_monitoring_viya.sh
 ```
 
-9. Deploy Prometheus to each SAS Viya namespace
+7. Deploy Prometheus to each SAS Viya namespace
 ```bash
 kubectl apply -n viya-one -f $USER_DIR/monitoring/prometheus-viya-one.yaml
 kubectl apply -n viya-two -f $USER_DIR/monitoring/prometheus-viya-two.yaml
```

Comment:
shouldn't we refer to **$VIYA_ONE_NS** and **$VIYA_TWO_NS** here instead **viya-one** and **viya-two**?

Reply:
@BryanEllington, please verify that these commands are correct. I don't think I changed them from your original README for this sample.